### PR TITLE
Increasing previous transactions count

### PR DESCRIPTION
**Pull request by irinatarn**

### CODE REVIEW

File: backend/queues/src/main/resources/application-int.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@ azure:
     app-service-principal-url: https://graph.microsoft.com/v1.0/servicePrincipals/${DFP_SP_ID}
     user-photo-url-template: https://graph.microsoft.com/beta/users/#user_id#/photo/$value
     timeout: PT5S
-    previous-transactions-count: 20
+    previous-transactions-count: 50
```
Comment (satyamamit) [inline backend/queues/src/main/resources/application-int.yml:93]: how did you make decision of 50 ?

**Review by satyamamit — CHANGES_REQUESTED** [review]

File: backend/queues/src/main/resources/application-int.yml (hunk 1 above)
Comment (irinatarn) [reply backend/queues/src/main/resources/application-int.yml:93]: It will give us nice velocity, no?
Didn't want to put 100 yet. Slowly increments

Comment (satyamamit) [reply backend/queues/src/main/resources/application-int.yml:93]: I mean did you evaluate how the size will be for 50 history transactions. we shouldn't be breaking LI team talk and again.

Comment (satyamamit) [reply backend/queues/src/main/resources/application-int.yml:93]: based on our analysis, I think 50 shouldn't increase the size of document by drastic numbers so we can go with 50.

**Review by satyamamit — APPROVED** [review]

**Pull request merged**